clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/controllers/EntityTagsController.java

```diff
@@ -42,7 +42,7 @@
   private final MessageSource messageSource;
   private final EntityTagsProvider tagProvider;
 
-  @Autowired(required = false)
+  @Autowired
```

Comment:
Are we sure this will work? How It seems like if `MessageSource` can't be constructed this won't startup.

Reply:
This was already effectively required as there's no default constructor; I tested injecting an unconstructed bean into this constructor and the application failed to start even before removing `required=false`.  So this will not cause startup to fail in any cases where it isn't already.  (And I'm actually not sure you could ever have a situation where `messageSource` hasn't been constructed as spring provides a default `DelegatingMessageSource` to fall back on.)